api/graphql/units/unit_serializers.py

```diff
@@ -3,5 +3,15 @@
 
 
 class UnitUpdateSerializer(UnitSerializer, PrimaryKeyUpdateSerializer):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.fields["tprek_id"].required = False
```

Comment:
How does the validation work with all of these set to not required?

Reply:
This case I don't see the problem when the tprek is the master (I actually don't know why we're editing the unit data in tvp :flushed: ) so this is just an extension to modify unit data.
Of course we might add some fields afterwards to unit level and so on but those optional marked fields are optional in gql update (create not even possible atm).